src/main/java/ch/njol/skript/expressions/ExprNumbers.java

```diff
@@ -55,6 +55,10 @@ public class ExprNumbers extends SimpleExpression<Number> {
 	static {
 		Skript.registerExpression(ExprNumbers.class, Number.class, ExpressionType.COMBINED,
 				"[(all [[of] the]|the)] (numbers|1¦integers|2¦decimals) (between|from) %number% (and|to) %number%");
+
+		ExprLoopValue.registerLoopValueHandler(ExprNumbers.class, (source, type) ->
+			source.mode == 1 && (type.equalsIgnoreCase("integer") || type.equalsIgnoreCase("int"))
```

Comment:
Also not sure if this one is needed, should be automatically handled with proper return type

Reply:
It's okay for this, but it does *not* work for
```vb
loop decimals between 20 and 30:
  broadcast loop-double # does not work
  broadcast loop-decimal # does not work
  broadcast loop-integer # does not work
  broadcast loop-number # works
```
Should we allow the usage of `loop-double` or `loop-decimal`, or do you think it's not worth it?

Reply:
I've updated this loop handler for just allowing `loop-decimal` since that isn't covered by checks (and we allow `loop decimals`)